package.json

```
@@ -2864,6 +2864,144 @@
 												]
 											}
 										]
+									},
+									{
+										"key": "languageId:python",
+										"name": "Python",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "m",
+														"name": "Format modified lines only",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													},
+													{
+														"key": "S",
+														"name": "+Format selection with",
+														"type": "command",
+														"command": "editor.action.formatSelection.multiple"
+													}
+												]
+											},
+											{
+												"key": "c",
+												"name": "+Execute",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "c",
+														"name": "Execute file in terminal",
+														"type": "command",
+														"command": "python.execInTerminal"
+													},
+													{
+														"key": "C",
+														"name": "Execute file in terminal",
+														"type": "command",
+														"command": "python.execInTerminal"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "I",
+														"name": "Sort imports",
+														"type": "command",
+														"command": "python.sortImports"
+													}
+												]
+											},
+											{
+												"key": "s",
+												"name": "+REPL",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "i",
+														"name": "Start REPL",
+														"type": "command",
+														"command": "python.startREPL"
+													},
+													{
+														"key": "l",
+														"name": "Send line/selection to terminal",
```

```suggestion
														"name": "Send line/selection to REPL",
```
Do you think it's better in this way?